vip-helpers/vip-query-monitor.php

```diff
@@ -65,7 +71,9 @@ public function process_db_object($id, wpdb $db)
 				$types[ $type ]['callers'][ $caller ]++;
 			}
 
-			$row = compact('caller', 'caller_name', 'stack', 'sql', 'ltime', 'result', 'type', 'component', 'trace');
+
+			$is_main_query = ( $request === $sql && ( false !== strpos( $stack, ' WP->main,' ) ) );
+			$row = compact('caller', 'caller_name', 'stack', 'sql', 'ltime', 'result', 'type', 'component', 'trace', 'has_main_query', 'is_main_query');
```

Comment:
`has_main_query` isn't defined until later.

Reply:
Nice catch! Thank you!